core/src/main/java/org/bitcoinj/core/AbstractBlockChain.java

```diff
@@ -134,14 +136,26 @@
 
     private final VersionTally versionTally;
 
-    /** See {@link #AbstractBlockChain(Context, List, BlockStore)} */
-    public AbstractBlockChain(NetworkParameters params, List<? extends Wallet> transactionReceivedListeners,
+    /**
+     * Constructs a BlockChain connected to the given list of listeners (wallets) and a store.
+     *
+     * @param params network parameters for this chain
+     * @param wallets list of listeners (wallets)
+     * @param blockStore where to store blocks
+     * @throws BlockStoreException if a failure occurs while storing a block
+     */
+    public AbstractBlockChain(NetworkParameters params, List<? extends Wallet> wallets,
                               BlockStore blockStore) throws BlockStoreException {
-        this(Context.getOrCreate(params), transactionReceivedListeners, blockStore);
```

Comment:
This change is not a JavaDoc fix.

Reply:
Well, I just renamed the parameter so it would have the same name in the two constructors. So it's mostly a javadoc fix.

Reply:
@schildbach It's your call, I can change the name back to `transactionReceivedListeners` and force push if you want.

Reply:
Ah, you're right.